tmt/cli.py

```diff
@@ -222,10 +222,19 @@ def main(click_contex, root, context, **kwargs):
 @click.option(
     '-S', '--skip', type=click.Choice(tmt.steps.STEPS), metavar='STEP',
     help='Skip given step(s) during test run execution.', multiple=True)
-@click.option(
-    '-e', '--environment', metavar='KEY=VALUE|@FILE', multiple='True',
-    help='Set environment variable. Can be specified multiple times. '
-         'The "@" prefix marks a YAML file to load.')
+@click.option('-e',
+              '--environment',
+              metavar='KEY=VALUE|@FILE',
+              multiple='True',
+              help='Set environment variable. Can be specified multiple times. '
+              'The "@" prefix marks a file to load (yaml or dotenv formats supported.')
+@click.option('-f',
```

Comment:
For now I'd recommend to omit the short option. Not sure how frequently this will be used. Plus this one already conflicts with `--force`.

Reply:
okay